index.js

```diff
@@ -306,6 +306,11 @@
     };
   };
 
+  var ValidDate = $.NullaryType(
+    'ValidDate',
+    function (x) { return $$typeEq('Date')(x) && !isNaN(x.getTime()) }
```

Comment:
For some reason I prefer to use `.valueOf()` rather than `.getTime()`. I can't justify this preference. ;)